src/devtools/client/shared/components/splitter/SplitBox.css

```diff
@@ -60,9 +60,13 @@
   pointer-events: none;
 }
 
-/* This disables splitter bar dragging when the user is 
+/* This disables splitter bar dragging when the user is
 in the console. Dragging/resizing should only work in the debugger
 or the inspector where the split console can be displayed.  */
 #toolbox.console .split-box.horz > .splitter {
   pointer-events: none;
 }
+
+.inspector-sidebar-splitter > .splitter {
+  background: var(--theme-splitter-color);
+}
```

Comment:
I recently added https://github.com/RecordReplay/devtools/blob/c825925daa9294c21239f076c06896996c51b440/src/devtools/client/themes/inspector.css#L46-L48, which does the same thing. One of them could be removed.

Reply:
I even remember it and yet mistakenly thought it _was_ the same file; will remove.